api/db/changelog/db.changelog-57-add-era-commons-completion-time.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+    xmlns="http://www.liquibase.org/xml/ns/dbchangelog/1.9"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog/1.9
+                    http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-1.9.xsd">
+  <changeSet author="blrubenst" id="db.changelog-57-add-era-commons-completion-time">
+    <addColumn tableName="user">
+      <column name="era_linked_nih_username" type="varchar(255)"/>
```

Comment:
Let's try to be consistent with a single module name prefix for all columns.

In the doc I'd proposed "eraCommons" for the prefix, which would lead to column names like:

era_commons_linked_nih_username
era_commons_link_expire_time

If we wanted to shorten the module name to "era_link" that's fine too – but needs to be consistent:

era_link_nih_username
era_link_expire_time

(My $.02 is that "era commons" is generally thought of as an atomic phrase, so it would feel a bit awkward to shorten to just "era link". I don't have super strong feelings on the name – but we do need to be internally consistent.)

Reply:
link vs linked is mirroring what FC/Shibboleth returns. I was just doing the prefix/module as `era`.